lib/lte_link_control/lte_lc.c

```diff
@@ -125,6 +125,20 @@ static const char nw_mode_preferred[] = "AT%XSYSTEMMODE=1,0,1,0";
 static const char nw_mode_fallback[] = "AT%XSYSTEMMODE=0,1,1,0";
 #endif
 
+#if defined(CONFIG_LWM2M_CARRIER) && !defined(CONFIG_GPS_USE_SIM)
+#if defined(CONFIG_BOARD_NRF9160_PCA20035NS)
```

Comment:
For reference, these are the commands that cannot be run during boot for Thingy:91:
https://github.com/NordicPlayground/fw-nrfconnect-nrf/blob/master/boards/arm/nrf9160_pca20035/board_nonsecure.c#L26

Currently they give a couple of error messages because BSD lib isn't enabled at that point in boot when carrier lib is used. I think we should add an ifdef for `CONFIG_BSD_LIBRARY_SYS_INIT` at that point in `board_nonsecure.c`.
Also, I suspect we need the COEX0 command as well to support newer Thingy revisions. @bjda can you confirm?

Reply:
Can confirm, the COEX0 command is needed for anything newer than 1.0.0